open-x-htb.js

```diff
@@ -361,7 +362,7 @@ function OpenXHtb(configs) {
       __profile = {
             partnerId: 'OpenXHtb',
             namespace: 'OpenXHtb',
-            statsId: 'OPNX',
+            statsId: 'OX',
```

Comment:
According to targetingKeys definition: https://github.com/indexexchange/open-x-adapter/blob/master-v2/open-x-htb.js#L383 and the given unit tests: https://github.com/indexexchange/open-x-adapter/blob/master-v2/spec/profile.spec.js#L94 , the __profile.statsId should be 'OX' instead of 'OPNX'.
I also checked some calls to https://securepubads.g.doubleclick.net/gampad/ads? on some client's page and it is indicating ix_ox_om and ix_ox_id are used.
Please confirm this change is correct. And please confirm ix_ox_om and ix_ox_id are used as the targeting keys in client's lineitem setup so it can be triggered correctly.